src/SarifWeb/Views/Home/Index.cshtml

```diff
@@ -225,7 +225,7 @@
                 For viewing and taking-action-on SARIF files within Visual Studio. Analysis results show up in the Error List, result details in a dockable tool window, and the results locations are overlayed on the original code (when available).
             </div>
             <div class="viewer-screenshot">
-                <img id="screenshot01" src="~/Images/Screenshots/VSViewer.thumbnail.png" index="0" coloration="dark" role="button" aria-labelledby="vsViewerCardTitle" alt="Visual Studio extension" />
+                <img id="screenshot01" src="~/Images/Screenshots/VSViewer.thumbnail.png" index="0" coloration="dark" role="button" aria-labelledby="vsViewerCardTitle" alt="Visual Studio extension" width="360" height="240" />
```

Comment:
Curious where the 360 value is coming from. I see 380 on the current site.

Reply:
this is set as the same as the original file size and the browser can resize according to the size of the window so we will see differently.